platform/ios/src/MGLPinAnnotationView.h

```diff
@@ -0,0 +1,12 @@
+#import <Mapbox/Mapbox.h>
+
+MGL_EXPORT
+
+@interface MGLPinAnnotationView : MGLAnnotationView
```

Comment:
Let’s reconsider the name of this class — in MapKit, the “pin” of [`MKPinAnnotationView`](https://developer.apple.com/documentation/mapkit/mkpinannotationview) refers to the pre-iOS 11 image they used (which was a skeuomorphic pin).  Our class is not visually a pin and is more closely aligned to [`MKMarkerAnnotationView`](https://developer.apple.com/documentation/mapkit/mkmarkerannotationview), if we’re intent on following MapKit naming conventions.

Reply:
Per our conversation, we're going to keep the name of this class the same for now.